security/vTPM-tests.py

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env python
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See LICENSE for more details.
+#
+# Copyright: 2021 IBM.
+# Author: Nageswara R Sastry <rnsastry@linux.ibm.com>
+
+import os
+from avocado import Test
+from avocado.utils import dmesg, genio, linux_modules
+
+
+class vTPM(Test):
+    """
+    vTPM tests for Linux
+    :avocado: tags=privileged,security,tpm
+    """
+    def setUp(self):
+        device_tree_path = "/proc/device-tree/vdevice/vtpm@30000003"
```

Comment:
Is this `30000003` always the case (in your environment)?

Reply:
@stefanberger Yes, the same number seen always on different systems.